water.mss

```diff
@@ -341,12 +341,15 @@
   [feature = 'landuse_basin'],
   [feature = 'waterway_dock'] {
     [zoom >= 0][way_pixels > 3000][way_pixels <= 768000],
-    [zoom >= 17] {
+    [zoom >= 14][way_pixels <= 768000][feature = 'natural_bay'],
+    [zoom >= 14][way_pixels <= 768000][feature = 'natural_strait'],
+    [zoom >= 17][way_pixels <= 768000] {
       text-name: "[name]";
       text-size: 10;
       text-wrap-width: 25; // 2.5 em
       text-line-spacing: -1.5; // -0.15 em
-      [way_pixels > 12000] {
+      [way_pixels > 12000],
+      [zoom >= 15][feature = 'natural_strait'] {
```

Comment:
Should 'natural_bay' also be rendered the same way at >= z15? (Doesn't need to be in this PR)

Reply:
I prefer making small and clear PRs if possible, please.